tsds-grafana-handler/tsds_grafana_handler.py

```diff
@@ -200,12 +200,11 @@ def bucket_size(size):
                         val = buckets.pop(0)
                 replace_val = str(bucket_size(val))
 
-                if len(indexes) > 0:
-                        alias = positions[indexes.pop(0)]
-                        swap  = aliases[alias]
-                        del aliases[alias]
-                        alias = alias.replace('$quantify', replace_val)
-                        aliases[alias] = swap
+                alias = positions[indexes.pop(0)]
+                swap  = aliases[alias]
+                del aliases[alias]
+                alias = alias.replace('$quantify', replace_val)
+                aliases[alias] = swap
```

Comment:
I think something actually got lost here, the if len(indexes) > 0 part. Be careful when doing cp that you are running from the latest version :)

If you can get this added, I think the rest looks good.